plugins/modules/ipa_host.py

```diff
@@ -74,7 +74,9 @@
     type: list
     elements: str
   state:
-    description: State to ensure.
+    description:
+    - State to ensure.
+    - V("present") and V("enabled") give the same results.
```

Comment:
This looks wrong to me. A `present` host can both be enabled or disabled.

Reply:
yes, but there are no such thing as 'enabling' host, because host is considered as 'enabled' if it has a keytab so to 'enable' a host you would need to do something completely different https://freeipa.readthedocs.io/en/latest/api/host_disable.html
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/identity_management_guide/host-disable#reenabling-hosts
and this module should not create keytab files on its own so I don't see a reason for having 'enabled' state other than to be an alias for 'present'

Reply:
I don't know FreeIPA at all, I only know about Ansible semantics, and what I wrote is what enabled/disabled usually means, and seems to be what the module also means (or better: intended do mean) when looking at the current implementation.

Reply:
Well, in that case I will keep it as it was before my changes